File: Develop/Backend/src/main/java/com/hientran/sohebox/sco/UserSCO.java
package com.hientran.sohebox.sco;

import com.fasterxml.jackson.annotation.JsonInclude;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@ToString
@AllArgsConstructor
@NoArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
@EqualsAndHashCode(callSuper = true)
public class UserSCO extends BaseSCO {
	private SearchNumberVO id;
	private SearchTextVO userName;
	private SearchTextVO firstName;
	private SearchTextVO lastName;
}

File: Develop/Frontend/src/app/scos/userSCO.ts
import { BaseSCO } from './baseSCO';
import { SearchNumber } from './core_sco/searchNumber';
import { SearchText } from './core_sco/searchText';

export class UserSCO extends BaseSCO {
  id: SearchNumber;
  userName: SearchText;
  firstName: SearchText;
  lastName: SearchText;
}
